### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by FireArrow108**
Remove unnecessary stuff
Comment

### CODE REVIEW

File: MineAdminWF/Main.vb [hunk 1]
```diff
@@ -422,16 +447,14 @@ It was only made for debugging purposes and expirements.", "Expiremental feature
 
     End Sub
 
-    Private Sub PluginsList_SelectedIndexChanged(sender As Object, e As EventArgs) Handles PluginsConfigList.SelectedIndexChanged
-
-    End Sub
-
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:428]: Remove unnecessary ```sub```

File: MineAdminWF/Main.vb [hunk 2]
```diff
@@ -351,17 +374,19 @@ It was only made for debugging purposes and expirements.", "Expiremental feature
...
     Private Sub ShellInput_KeyDown(sender As Object, e As KeyEventArgs) Handles ShellInput.KeyDown
+        'Send shell command
         If (e.KeyCode = Keys.Enter) Then
             ShellOutput.Text = Shell(ShellInput.Text)
         End If
     End Sub
 
     Private Sub ManageTabControl_SelectedIndexChanged(sender As Object, e As EventArgs) Handles ManageTabControl.SelectedIndexChanged
-        'Loading the config
+        'Load the config for the Edit tab
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:389]: Made it more clear

File: MineAdminWF/Main.vb [hunk 3]
```diff
@@ -1,12 +1,14 @@
-﻿Imports System.IO
+﻿Imports System.IO   'Needed when reading config files and installing software
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:1]: Just in case if someone wants to remove it

File: MineAdminWF/Main.vb [hunk 4]
```diff
@@ -59,17 +64,19 @@ Public Class Form
         SnapshotsSwitch.Enabled = True
         SnapshotSelector.Visible = False
         SnapshotSelector.Enabled = False
-
     End Sub
 
     Private Sub CreateServerConfirm_Click(sender As Object, e As EventArgs) Handles CreateServerConfirm.Click
+        ''''''''''''''''''''''''''''''''''''''
+        '       Create server sequence       '
+        ''''''''''''''''''''''''''''''''''''''
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:72]: How to mark important code parts

File: MineAdminWF/Main.vb [hunk 5]
```diff
@@ -133,19 +140,14 @@ Currently supporting 1.12.2, 1.14.4, 1.15.2 and 1.16.3", "Error", MessageBoxButt
...
-        If VersionSelector.Text = "1.16.3" Or VersionSelector.Text = "1.15.2" Or VersionSelector.Text = "1.14.4" Or VersionSelector.Text = "1.12.2" Then
-            If OkEULA.Checked = True Then
-                CreateServerConfirm.Enabled = True
-            Else
-                CreateServerConfirm.Enabled = False
-            End If
-        Else
-            CreateServerConfirm.Enabled = False
-        End If
-    End Sub
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:146]: I don't want to limit the user to just 4 versions, even more when _Custom Version_s came in

File: MineAdminWF/Main.vb [hunk 6]
```diff
@@ -133,19 +140,14 @@ Currently supporting 1.12.2, 1.14.4, 1.15.2 and 1.16.3", "Error", MessageBoxButt
...
 
     Private Sub StartButton_Click(sender As Object, e As EventArgs) Handles StartButton.Click
+        '''''''''''''''''''''''''''''''''''''''
+        '       Server startup sequence       '
+        '''''''''''''''''''''''''''''''''''''''
+        'TODO:
+        '   User defines RAM
+        '   User defines JAR path
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:150]: In my opinion, these are **very** important

File: MineAdminWF/Main.vb [hunk 7]
```diff
@@ -158,18 +160,22 @@ Currently supporting 1.12.2, 1.14.4, 1.15.2 and 1.16.3", "Error", MessageBoxButt
         End If
     End Sub
 
-    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles StopButton.Click
```
Comment (FireArrow108) [inline MineAdminWF/Main.vb:161]: Why is it named ```Button2```

File: MineAdminWF/Main.vb (hunk 2 above)
Comment (Verarr) [reply MineAdminWF/Main.vb:389]: Just random double clicking, huh?

**Review by Verarr — APPROVED** [review]
OK!

**Pull request merged**